hazelcast/src/main/java/com/hazelcast/cache/impl/operation/CachePutAllOperation.java

```diff
@@ -87,7 +87,10 @@ public void run()
             Data key = entry.getKey();
             Data value = entry.getValue();
             CacheRecord backupRecord = cache.put(key, value, expiryPolicy, callerUuid, completionId);
```

Comment:
Shouldn't we send new put to backups independent of the expiry? 
Otherwise retrieved cache stats can be false from `CacheStatistics#getCachePuts` or `CacheStatistics#getLastAccessTime`

Reply:
With expire-on-create the result of `cache.put` is `null` and the value is not put on the primary either. The same already happens in `CachePutOperation`, if `backupRecord` is `null` then no backup is sent: https://github.com/hazelcast/hazelcast/blob/456a6366d087957f5e7e73827a686e4a4c8a00bc/hazelcast/src/main/java/com/hazelcast/cache/impl/operation/CachePutOperation.java#L83

With regards to statistics effects, when configured with expire-on-create `cachePuts` are not expected to be incremented as the value is not actually put in the cache. See also [this TCK test](https://github.com/jsr107/jsr107tck/blob/e9409a8e10b507cf7ff400fee2f4e5f4b7316f9b/cache-tests/src/test/java/org/jsr107/tck/management/CacheMBStatisticsBeanTest.java#L849-L867) that enforces this assertion.